container/start.sh

```diff
@@ -37,7 +38,7 @@ start_karapace_registry(){
     "group_id": "${KARAPACE_REGISTRY_GROUP_ID:-$KARAPACE_REGISTRY_GROUP_ID_DEFAULT}",
     "master_eligibility": ${KARAPACE_REGISTRY_MASTER_ELIGIBITY:-$KARAPACE_REGISTRY_MASTER_ELIGIBITY_DEFAULT},
     "topic_name": "${KARAPACE_REGISTRY_TOPIC_NAME:-$KARAPACE_REGISTRY_TOPIC_NAME_DEFAULT}",
-    "compatibility": "FULL",
+    "compatibility": "${KARAPACE_REGISTRY_COMPATIBILITY:-${KARAPACE_REGISTRY_COMPATIBILITY_DEFAULT}",
```

Comment:
```suggestion
    "compatibility": "${KARAPACE_REGISTRY_COMPATIBILITY:-$KARAPACE_REGISTRY_COMPATIBILITY_DEFAULT}",
```

You have syntax error here, but other than that the contribution makes sense

Reply:
Thanks, fixed now